js/ElementRendererProvider.js

```diff
@@ -452,6 +428,8 @@ export default class ElementRendererProvider {
               (arrowLeft: any).style.visibility = 'hidden';
               (arrowRight: any).style.visibility = 'hidden';
             }
+            // Set up card reference for carousel
+            cards = carousel.getElementsByClassName('lp-json-pollock-layout');
```

Comment:
A card can also be a simple element, meaning a button or text element.
In that case, it will not be on the cards list and will be ignored.

I suggest taking a different approach - a card is a direct child of the carousel element.

WDYT?

Reply:
Good point, I will get it to use the child elements of the carousel. We won't have anything added there that is not a card, correct?

Reply:
Carousel can have layouts and elements as children. ad card is another name we usually call to a vertical layout